umodbus/asynchronous/serial.py

```diff
@@ -0,0 +1,397 @@
+                                     request_data=request_data,
+                                     values=values,
+                                     signed=signed)
+        if task is not None:
+            await task
+
+    async def send_exception_response(self,
+                                      slave_addr: int,
+                                      function_code: int,
+                                      exception_code: int,
+                                      request: Optional[AsyncRequest] = None) \
+            -> None:
+        """
+        Asynchronous equivalent to Serial.send_exception_response
+        @see Serial.send_exception_response for common (leading) parameters
+
+        :param      request:     Ignored; kept for compatibility
+                                 with AsyncRequest
+        :type       request:     AsyncRequest, optional
+        """
+
+        task = super().send_exception_response(slave_addr=slave_addr,
+                                               function_code=function_code,
+                                               exception_code=exception_code)
+        if task is not None:
+            await task
+
+    async def get_request(self,
+                          unit_addr_list: Optional[List[int]] = None,
+                          timeout: Optional[int] = None) -> \
+            Optional[AsyncRequest]:
+        """@see Serial.get_request"""
+
+        req = await self._uart_read_frame(timeout=timeout)
+        req_no_crc = self._parse_request(req=req,
+                                         unit_addr_list=unit_addr_list)
+        try:
+            if req_no_crc is not None:
+                return AsyncRequest(interface=self, data=req_no_crc)
+        except ModbusException as e:
+            await self.send_exception_response(slave_addr=req[0],
+                                               function_code=e.function_code,
+                                               exception_code=e.exception_code)
+
+
+async def _async_send(device: Union[AsyncRTUServer, AsyncSerial],
+                      modbus_pdu: bytes,
+                      slave_addr: int) -> None:
+    """
+    Send modbus frame via UART asynchronously
+
+    Note: This is not part of a class because the _send()
+    function exists in CommonRTUFunctions, which RTUServer
+    extends. Putting this in a CommonAsyncRTUFunctions class
+    would result in a rather strange MRO/inheritance chain,
+    so the _send functions in the client and server just
+    delegate to this function.
+
+    :param      device:     The self object calling this function
+    :type       device:     Union[AsyncRTUServer, AsyncSerial]
+
+    @see CommonRTUFunctions._send
+    """
+
+    serial_pdu = device._form_serial_pdu(modbus_pdu, slave_addr)
+    send_start_time = 0
+
+    if device._ctrlPin:
+        device._ctrlPin(1)
+        # wait 1 ms to ensure control pin has changed
+        await asyncio.sleep(1/1000)
```

Comment:
```suggestion
        await asyncio.sleep(1 / 1000)
```

missing whitespace around arithmetic operator